src/main/java/sh/platform/config/Route.java

```diff
@@ -0,0 +1,65 @@
+package sh.platform.config;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Optional;
+
+public class Route {
```

Comment:
Is introducing this an API break?  I'm fine with doing so, but concerned if it becomes an API break in Java.  (I don't know how Java handles such things.)

Reply:
Yeap, that is break compatibility.
I'll need to put that information on the release.

Reply:
Per SemVer, that means a 3.x release here.  I'm a bit concerned about getting out of sync version wise with the other libraries.  (That said, I'm tempted to have a Route class in PHP, too.  Go already has one.  But that is, as I said, an API break.)